addons/gearinfo/XtdGearModels.hpp

```diff
@@ -201,10 +198,6 @@ class XtdGearModels
 			label = "CBRN Suit";
 			author = "Bohemia Interactive";
 			options[] = {"camo"};
-			class camo : CamoBase
-			{
-				values[] = {"WHT", "BLU", "M81", "MTP", "Tropic", "AAF", "EAF"};
```

Comment:
We should keep this entry, otherwise all camos will be displayed for that model

Reply:
An other option would be to check if a variation exists with each value to avoid returning non existant value

Reply:
I already had the second approach implemented in a commit from the draft PR, so I picked the relevant parts from it.